testing/etckeeper/etckeeper.trigger

```diff
@@ -0,0 +1,11 @@
+#!/bin/sh
+
+# we are in trigger context so the apk database
+# contains info before the change takes effect
+/etc/etckeeper/pre-install.d/10packagelist
+
+LOCK="/lib/apk/db/lock"
+exec 200>"$LOCK"
+
+# wait for $LOCK before commit
+(flock --timeout 180 200 && etckeeper post-install) &
```

Comment:
Is this trigger script hack something we get rid of after https://github.com/alpinelinux/apk-tools/pull/2 is merged?